src/components/core/DpObscure.vue

```diff
@@ -1,20 +1,13 @@
 <template>
   <span
-    :class="obscureClass"
+    class="u-obscure"
     :title="Translator.trans('obscure.title')">
     <slot />
   </span>
 </template>
 
 <script>
 export default {
-  name: 'DpObscure',
-
-  computed: {
-    obscureClass () {
-      return (hasPermission('feature_obscure_text')) ? 'u-obscure' : ''
-    }
```

Comment:
I fear this one is a bit tricky. 
If we simply remove the permission here, we loose the benefit of the check.
At the moment it behave like this: 
If the permission is true, the Text gets a black background. But if not, It comes along like a regular text.
After backing the class in, we loose the effect of displaying the regular text.
I think we have to add a prop for the permission here. But:
the changes in `demosplan-core` won't be simple. DpObscure is implemented in a twig-filter and is registered globaly because some text wich may uses dpObscure comes from the Database.

Reply:
> If the permission is true, the Text gets a black background. But if not, It comes along like a regular text.
After backing the class in, we loose the effect of displaying the regular text.

That was my first thought as well. But under which circumstance would we want to show the obscured text at all? 

Within admin editor instances, users still see the obscured texts in non-edit-mode

![image](https://user-images.githubusercontent.com/40452344/214854275-ceebee55-402d-4e72-a14d-3468a929c100.png)

as well as edit-mode:

![image](https://user-images.githubusercontent.com/40452344/214854352-16715eb9-8349-41c6-befb-59ae8dc2f36d.png)

When exporting statements, the whole text is passed through the demosplan/DemosPlanCoreBundle/Logic/EditorService.php:185, replacing everything with █ . for non-obscured exports, the existence of that class within the markup will have no effect as it is rendered in non-html contexts.

Reply:
:thinking:  Ok. You may be right. But after this changes are installed in demosplan-core these Changes I would please You @ahmad-demos to tell the testers to have a closer look at that obscure-feature. :pray:

Reply:
+1 for that!